app/controllers/documents_controller.rb

```diff
@@ -83,7 +85,8 @@ def resource_params
                                      :document_category_id,
                                      :url,
                                      :available_to_public,
-                                     locality_ids: []
+                                     locality_ids: [],
```

Comment:
in the interests of consistency with the other controllers, how about this instead:
`
              { building_ids: [], person_ids: [], locality_ids: [] }`